scripts/wsrep_sst_rsync.sh

```diff
@@ -340,8 +361,12 @@ $SILENT
     path = $WSREP_SST_OPT_DATA
 [$MODULE-log_dir]
     path = $WSREP_LOG_DIR
+[$MODULE-data_dir]
+    path = $INNODB_DATA_HOME_DIR
 EOF
 
+[ "$(whoami)" != root ] || sed -i '/read only = no/s/.*/&\nuid = root\ngid = root\n/' "$RSYNC_CONF"
```

Comment:
Is this then rsync user, mariadb user or user running mariadb process, again I think we do not need any special case for root.

Reply:
I can remove root support, but then several mtr tests can fail if mtr running under the root.